tools/upload_tables.sh

```diff
@@ -89,6 +100,21 @@ check_dataset_or_table() {
         fi
 }
 
+check_uuid_in_metadata() {
+	local meas_uuid="$1"
+	local bq_metadata_table="$2"
+	local query
+	local query_result
+
+	query="SELECT COUNT(*) FROM \`${bq_metadata_table}\` WHERE id = '${meas_uuid}'"
+	echo "${query}"
+	query_result=$(bq query --use_legacy_sql=false --project_id="${GCP_PROJECT_ID}" --format=csv "${query}" | tail -n 1)
+	echo "${query_result}"
+	if [[ "${query_result}" -eq 0 ]]; then
```

Comment:
If this is a number comparison, we don't need the double quotes.  Or, to compare strings, we can do:
	if [[ "${query_result}" == "0" ]]; then